src/button/index.js

```diff
@@ -5,7 +5,7 @@ import { compose, defaultProps } from 'recompose';
 import { body } from '../theme/fonts';
 import * as colors from '../theme/colors';
 
-const strokeColor = colors.C300
+const strokeColor = colors.N0
```

Comment:
Any reason this is white by default now?

Reply:
Switched it to white because it seemed to be the button colour that was more consistent throughout the existing site (but not necessarily the style guide). The style guide has the primary button as something we never ended up using?